packages/wrangler/src/dev/local.tsx

```diff
@@ -251,12 +251,18 @@ function useLocalWorker({
         process.stdout.write(data);
       });
 
+      // parse the node inspector url (which may be received in chunks) from stderr
+      let foundInspectorUrl = false;
+      const stderrData: string[] = [];
       local.current.stderr?.on("data", (data: Buffer) => {
+        if (foundInspectorUrl) return;
```

Comment:
Currently we don't bother to stop listening when we have found the inspector URL, but this seems like a wise thing to do.

Instead of using a flag for this (and continuing to listen), how about we unbind the event listener once we have got what we want?